docs/install/knative-with-operators.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "Installing Knative with Knative Operators"
+weight: 10
+type: "docs"
+---
+
+# Knative installations with Knative Operators
+
+As of v0.10.0, Knative started to release Knative operators as tools to install, configure and manage Knative. This
+guide will explain how to install and uninstall Knative with Knative operators.
+
+There are two major components in Knative: Serving and Eventing. There are two operators: [Knative Serving Operator](https://github.com/knative/serving-operator)
+and [Knative Eventing Operator](https://github.com/knative/eventing-operator), respectively dedicated to each of them. Please make sure you have set up a Kubernetes
```

Comment:
These sentences read a little bit choppy (probably because you've got 4 things that are related to each other in a matrix-y way, but text is linear).

What about turning it around and emphasizing the operator first?

> Each component in Knative has a separate operator to install and configure the components. This means that there is a [Serving operator] and an [Eventing operator], and you can choose to install one or both independently.

Reply:
done.